modules/contact-form/grunion-contact-form.php

```diff
@@ -848,11 +848,13 @@ public function get_post_content_for_csv_export( $post_id ) {
 	 * @param int $post_id Id of the post to fetch meta data for.
 	 *
 	 * @return mixed
-	 *
-	 * @codeCoverageIgnore - No need to be covered.
 	 */
 	public function get_post_meta_for_csv_export( $post_id ) {
-		return get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md                  = get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md['feedback_date'] = get_the_date( 'Y-m-d H:i:sT', $post_id );
```

Comment:
```suggestion
		$md['feedback_date'] = get_the_date( DATE_RFC3339, $post_id );
```

I'd rather use 3339. `Y-m-D H:i:sT` vs RFC3339 would be:
`"2010-06-24 20:37:48CDT"` vs `"2010-06-24T20:37:48-05:00"`.

`CDT` and `CST` could mean Central Daylight Time or Central Standard Time which is UTC-5/-6 respectively. It can also mean Cuba Daylight Time or Cuba Standard Time which is UTC-4/-6.

PHP will output the abbreviation -- try `(new DateTime() )->setTimeZone(new DateTimeZone('America/Chicago'))->format('T');` vs `(new DateTime() )->setTimeZone(new DateTimeZone('America/Havana'))->format('T');`

Reply:
u r correct, sir!-) hadn't thought of that, i'll fix